common/source/docs/common-arkflow.rst

```diff
@@ -0,0 +1,53 @@
+.. _common-arkflow:
+
+================================
+ARK Flow Open Source Optical Flow and Distance Sensor
+================================
+
+[copywiki destination="copter,plane"]
+
+The `ARK Flow Open Source Optical Flow and Distance Sensor <https://arkelectron.com/product/ark-flow/>`__ is an open source optical flow sensor including a Broadcom AFBR lidar which uses the CAN protocol to communicate with the autopilot.  This can be used to improve horizontal position control especially in GPS denied environments.
+
+.. image:: ../../../images/arkflow/ark_flow.jpg
+   :target: ../_images/arkflow/ark_flow.jpg
+   :width: 450px
+
+..  youtube:: SAbRe1fi7bU
+    :width: 100%
+
+Where to Buy
+------------
+
+The sensor is available from `ARK Electronics <https://arkelectron.com/product/ark-flow/>`__
+
+Connection to Autopilot
+-----------------------
+
+- The ARK Flow is connected to the CAN bus using a Pixhawk standard 4 pin JST GH cable.
+- Multiple sensors can be connected by plugging additional sensors into the ARK Flow's second CAN connector.
+- The recommended mounting orientation is with the connectors on the board pointing towards back of vehicle, as shown in the following picture.
+
+.. image:: ../../../images/arkflow/ark_flow_orientation.png
+   :target: ../_images/arkflow/ark_flow_orientation.png
+   :width: 450px
+
+- Connect the sensor to the autopilots' CAN port (if using a Cube note that pre-Apr 2019 boards have CAN1 and CAN2 swapped)
```

Comment:
Just the labels are screen printed backwards. Maybe leave this tid bit to the manufacturer docs rather than these docs?

Reply:
I copied this from the Here Flow docs. We can remove this line.